crates/swc_ecma_parser/src/parser/object.rs

```diff
@@ -51,6 +51,23 @@ impl<I: Tokens> Parser<I> {
         .parse_with(|p| {
             let start = cur_pos!(p);
 
+            // The lexer marks keywords with escapes as errors, but unlike in
+            // the general case, it's OK here as a property name. We turn the
+            // error back into a token
+            if let Some(err_tok @ Token::Error(e)) = p.input.cur() {
```

Comment:
This is the wrong direction. We should adjust `ctx` to ignore the error at the first place.

Reply:
If by "adjusting ctx" you mean adding `with_ctx({ in_property_name: true, ... })` at the call site of a parser, I had initially thought it could have been too coarse... I need to try again.

Reply:
I think adding a field for ignoring escapes in keywords to the context type is the best solution